### PR TITLE
Update doc and fix py3 support

### PR DESCRIPTION
**Pull request by fortran01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,3 +198,9 @@ Pro-Tip: If you already use Tidal Migrations [Ansible Tower integration script](
 cd ansible-tower-integration
 ./tidal_inventory.py | jq -r '.servers.hosts[]' > path/to/hosts
 ```
+
+### Known issues
```
Comment (pshevtsov) [inline README.md:202]: What is the issue? Why may it happen?

File: unix/runner [hunk 2]
```diff
@@ -57,23 +59,57 @@ def cpu_name(proc):
...
+    def v2_runner_on_unreachable(self, result):
+        host = result._host
+        self.host_unreachable[host.get_name()] = result
+
+    def v2_runner_on_ok(self, result, *args, **kwargs):
+        """Print a json representation of the result.
+
+        Also, store the result in an instance attribute for retrieval later
+        """
+        # host = result._host
```
Comment (pshevtsov) [inline unix/runner:84]: No need to comment old code, we can retrieve it from git history if needed.

File: unix/runner [hunk 3]
```diff
@@ -57,23 +59,57 @@ def cpu_name(proc):
...
 
         TOTAL_RESULTS['servers'].append({
             'host_name': facts['ansible_hostname'],
             'fqdn': facts['ansible_fqdn'],
             'ip_addresses': facts['ansible_all_ipv4_addresses'] + facts['ansible_all_ipv6_addresses'],
             'ram_allocated_gb': ram_allocated_gb(facts),
             'ram_used_gb': ram_used_gb(facts),
-            'storage_allocated_gb': storage_allocated_gb(facts),
-            'storage_used_gb': storage_used_gb(facts),
+            # 'storage_allocated_gb': storage_allocated_gb(facts),
```
Comment (pshevtsov) [inline unix/runner:97]: Why not gathering storage use info?

**Review by pshevtsov — CHANGES_REQUESTED** [review]

**Comment by fortran01** [comment]
Clarification: I was just able to confirm it works with RHEL7, py2.7 and the same Ansible version baked into requirements.txt :thumbsup_all:

I will confirm py3 with venv later.

File: README.md [hunk 4]
```diff
@@ -148,6 +148,10 @@ CPU name
 4. `pip install -r requirements.txt`
 5. [Add your SSH key to `ssh-agent`](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#adding-your-ssh-key-to-the-ssh-agent) or have your SSH identity file available.
 
+### Known versions to work
```
Comment (fortran01) [inline README.md:151]: @pshevtsov WDYT?

**Comment by dcolebatch** [comment]
Are we going to abandon this PR now @fortran01 ?

**Comment by fortran01** [comment]
> Are we going to abandon this PR now @fortran01 ?

I am going to re-test `py3` on RHEL7 and sync this with the latest changes in `master`.

**Comment by prees1** [comment]
Closing this PR @fortran01. @pshevtsov has recently made some merges that cover the goal here. Here are @pshevtsov recent related PRs that are merged, https://github.com/tidalmigrations/machine_stats/pull/20, https://github.com/tidalmigrations/machine_stats/pull/19, https://github.com/tidalmigrations/machine_stats/pull/17

If anything on this is PR here is still needed feel free to reopen or let me know.

**Pull request closed without merging**